lldb/source/Plugins/Protocol/MCP/MCPError.h

```diff
@@ -30,4 +31,17 @@ class MCPError : public llvm::ErrorInfo<MCPError> {
   int64_t m_error_code;
 };
 
+class UnsupportedURI : public llvm::ErrorInfo<UnsupportedURI> {
```

Comment:
According to the MCP spec, there are 2 error codes for resources:

* Resource not found: -32002
* Internal errors: -32603

See https://modelcontextprotocol.io/specification/2025-06-18/server/resources#error-handling

Should we use those here?